src/backend.js

```diff
@@ -61,18 +67,32 @@ export default {
         return true
       }
       alert(response.statusText)
-      console.log(response.data)
+      console.debug(response.data)
       return null
     })
   },
 
   getRandomImages () {
-    return $axios.get(`image/random`)
+    let campaignId = localStorage.getItem('currentCampaignId') || null
+    return $axios.get(`image/random/` + campaignId)
       .then(response => response.data)
   },
 
   getVoteCount () {
     return $axios.get(`vote/count`)
       .then(response => response.data)
+  },
+
+  getNextCampaign () {
+    return $axios.get(`campaign/next`)
+      .then(function (response) {
+        if (response.status === 201) {
+          localStorage.setItem('currentCampaignId', response.data.id)
+          return response.data
+        }
+        console.debug('Default campaign selected')
+        localStorage.setItem('currentCampaignId', 1)
+        return { id: 1 }
```

Comment:
Do you think we should also use the default campaign in case of an exception?

Reply:
As long as it's properly logged . . . yes.